reference/architecture/index.md

```diff
@@ -128,8 +128,11 @@ By default Spinnaker binds ports according to the following table
 | Fiat        | 7003              |
 | Front50     | 8080              |
 | Gate        | 8084              |
+| Gate x509   | 8085              |
 | Halyard     | 8064              |
 | Igor        | 8088              |
 | Kayenta     | 8090              |
 | Orca        | 8083              |
 | Rosco       | 8087              |
+| Redis       | 6379              |
+| MariaDB     | 3306              |
```

Comment:
Should this reference `MySQL` instead?  From [the docs](https://www.spinnaker.io/setup/productionize/persistence/orca-sql/#mariadb) it looks like it's a pretty specific use case to use the MariaDB connector to connect to AWS and most people will just be using a MySQL installation.

Reply:
Yes. Could say either/or but to me MySQL and Maria are analogous so, +1. Will update